src/main/java/touro/GameOfLife/Grid.java

```diff
@@ -5,11 +5,7 @@
 
 public class Grid {
     private Direction[] directions = Direction.values();
-    private final HashSet<Cell> liveCells = new HashSet<>();
-
-    //TODO: find a reasonable height and width
-    public static final int HEIGHT = 20;
-    public static final int WIDTH = 20;
+    private HashSet<Cell> liveCells = new HashSet<>();
```

Comment:
I removed final from here because I thought it did not make sense to be final. It keeps changing which means it's anything but final

Reply:
good point